packages/grafana-data/src/transformations/transformers/formatString.ts

```diff
@@ -0,0 +1,90 @@
+import { map } from 'rxjs/operators';
+
+import { Field, FieldType } from '../../types';
+import { DataTransformerInfo } from '../../types/transformations';
+
+import { DataTransformerID } from './ids';
+
+export enum FormatStringOutput {
+  UpperCase = 'Upper Case',
+  LowerCase = 'Lower Case',
+  FirstLetter = 'Capitalize First Letter',
+  EveryFirstLetter = 'Capitalize First Letter Of Every Word',
```

Comment:
```suggestion
  TitleCase = 'Capitalize First Letter',
  SentanceCase = 'Capitalize First Letter Of Every Word',
```
...

also is this a place for "Trim"?

Reply:
Think this is resolved. Trim was also added